src/components/select-project/selectProject.html

```diff
@@ -0,0 +1,93 @@
+<form class="form-horizontal config-form">
+    <div class="form-group">
+        <label class="col-sm-4 control-label" for="project">Add to Project</label>
+      <div class="col-sm-8">
+        <ui-select ng-model="$ctrl.selectedProject">
+          <ui-select-match placeholder="Project name">
+            {{$select.selected | displayName}}
+          </ui-select-match>
+          <ui-select-choices repeat="project in $ctrl.projects | searchProjects : $select.search">
+            <span ng-bind-html="project | displayName | highlightKeywords : $select.search"></span>
+                <span ng-if="project | displayName : true" class="small text-muted">
+                  <span ng-if="project.metadata.name">&ndash;</span>
+                  <span ng-bind-html="project.metadata.name | highlightKeywords : $select.search"></span>
+                </span>
+          </ui-select-choices>
+        </ui-select>
+      </div>
+    </div>
+</form>
+
+<ng-form name="$ctrl.forms.createProjectForm" class="form-horizontal"
```

Comment:
The top form is selectProject whereas the bottom form is createProject.  Did change both to `ng-form`